v1.1/sql-statements.md

```diff
@@ -89,15 +89,31 @@ Statement | Usage
 
 Statement | Usage
 ----------|------------
-[`CANCEL QUERY`](cancel-query.html) | Cancel a running SQL query.
```

Comment:
@sploiselle, I integrated these updates as part of https://github.com/cockroachdb/docs/pull/1971, so you can just remove changes to this file from this PR.

Reply:
Also, after removing that file, is there any more work needed here? It'd be great if you could merge this so some follow-up PRs in progress could pass TC.